### DISCUSSION TITLE
New chapter styles

### DISCUSSION THREAD

**Discussion by catarinarurbano** [joaomlourenco/novathesis#337] [Q&A] [answered]
First of all, thank you so much for the awesome work on this template!

My question is: How to add more chapter styles from the memoir class? [ChapterStyles](https://mirrors.up.pt/pub/CTAN/info/latex-samples/MemoirChapStyles/MemoirChapStyles.pdf)

Thanks in advance

**Comment by joaomlourenco** [comment, marked as answer]
There is a folder `NOVAthesis-files/ChapStyles` with a file for each chapter style.  Some are very very simple… some are more complex.  Follow the examples…  :)

**Reply by catarinarurbano** [reply]
Thank you very much for your quick reply.

One more thing I noticed, when using \gls{} for acronyms, sometimes they appear in the format 'full name (acronym)' (for example, having three acronyms in a row, the first two appear with the description and the last one only as the acronym) and sometimes just the acronym itself. Is it possible to define for all cases that only the acronym appears, without its meaning before?

Thanks in advance. 
Have a good weekend.

**Reply by joaomlourenco** [reply]
You should not "reuse" a question for a different question.  Please create a new one (with this text) and I will reply there, ok?  Thank you.

Also, if you are satisfied with the answer to the original question, please select the "Mark as answer" below the answer. Thanks.

**Comment by joaomlourenco** [comment]
@catarinarurbano 

If the chapter style is provided by memoir itself, just duplicate `veelo.ldf` and replace
```latex
\chapterstyle{veelo}
```
with
```latex
\chapterstyle{your_desired_style}
```

If the chapter style is provided by an external package, e.g., [FncyChap](https://www.ctan.org/tex-archive/macros/latex/contrib/fncychap), also duplicate `veelo.ldf` and replace
```latex
\chapterstyle{veelo}
```
with
```latex
\RequirePackage[Style]{FncyChap}
```
where style is one of _Sonny, Lenny, Glenn, Conny, Rejne, Bjarne, PetersLenny, Bjornstrup_.

In both cases, use the (new) filename as the chapter style in `1_novathesis.tex`.

**Reply by catarinarurbano** [reply]
Thank you so much, professor! Have a wonderful weekend!
